docs/03-tutorials/00-api-exposure/apix-04-expose-and-secure-workload-jwt-istio-ext-auth.md

```diff
@@ -0,0 +1,145 @@
+---
+title: Expose and secure a workload with Istio external auth
+---
+
+This tutorial shows how to expose workload using VirtualService and secure it using Istio authorization policy together with Istio RequestAuthentication based on JWT token
+
+## Prerequisites
+
+To follow this tutorial, use Kyma 2.0 or higher.
+
+This tutorial is based on a sample HttpBin service deployment and a sample Function. To deploy or create one of those, follow the [Create a workload](./apix-02-create-workload.md) tutorial.
+
+## Expose your workload using VirtualService
+
+1. Export environment variable with your namespace:
+
+   ```shell
+   export NAMESPACE={NAMESPACE} # e.g. default
```

Comment:
in pre-reqs deploy workload is mentioned where ns was already exported. no need to repeat that